cmake/common/helpers_common.cmake

```diff
@@ -251,8 +251,9 @@ function(find_dependencies)
         else()
           continue()
         endif()
-      elseif(library MATCHES "\\$<.*Qt6::EntryPointPrivate>" OR library MATCHES
-                                                                "\\$<.*Qt6::QDarwinBluetoothPermissionPlugin>")
+      elseif(library MATCHES "\\$<.*Qt6::EntryPointPrivate>"
+             OR library MATCHES
+                "\\$<.*Qt6::QDarwin(Bluetooth|Calendar|Camera|Contacts|Location|Microphone)PermissionPlugin>")
```

Comment:
It's probably safe to use `Qt6::Darwin.+PermissionPlugin` because that's specific enough.

Reply:
Changed